src/granite/query/builder.cr

```diff
@@ -75,19 +75,13 @@ class Granite::Query::Builder(Model)
     and(stmt: stmt, value: value)
   end
 
-  def and(**matches)
-    and(matches)
-  end
-
-  def and(matches)
-    matches.each do |field, value|
-      and(field: field.to_s, operator: :eq, value: value)
-    end
+  def and(field : (Symbol | String), operator : Symbol, value : Granite::Columns::Type)
+    @where_fields << {join: :and, field: field.to_s, operator: operator, value: value}
 
     self
   end
 
-  def and(field : (Symbol | String), operator : Symbol, value : Granite::Columns::Type)
+  def and(field : (Symbol | String), operator : Symbol, value : Array(String | Int32 | Int64))
```

Comment:
I still don't think this should be required.

Reply:
you are right! Removed it